README.md

```diff
@@ -94,7 +92,15 @@ omit it if you want to install in the current working directory):
 npm install standard -g
 ```
 
-After you've done that you should be able to use the `standard` program. The
+Or, you can run this command to install `standard` locally, for use in your module:
+
+```bash
+npm install standard
+```
+
+## Usage
+
+After you've installed `standard`, that you should be able to use the `standard` program. The
```

Comment:
"After you've installed `standard`, you should be able to use the `standard` program"